### PR TITLE
feat: Allow ‘Grid Cell’ to render a custom HTML element

### PR DESCRIPTION
**Pull request by VincentSmedinga**
And update the approach on ‘Spotlight’.

### CODE REVIEW

File: packages/react/src/Spotlight/Spotlight.test.tsx [hunk 1]
```diff
@@ -48,4 +48,10 @@ describe('Spotlight', () => {
 
     expect(component).toHaveClass('amsterdam-spotlight--green')
   })
+
+  it('renders a custom tag', () => {
+    render(<Spotlight as="article" />)
```
Comment (dlnr) [inline packages/react/src/Spotlight/Spotlight.test.tsx:53]: Spotlight should be named in the PR title for future reference.

**Review by dlnr — APPROVED** [review]

File: packages/react/src/Spotlight/Spotlight.test.tsx (hunk 1 above)
Comment (VincentSmedinga) [reply packages/react/src/Spotlight/Spotlight.test.tsx:53]: Spotlight already supported this and its API hasn’t changed. What would you like to mention in the PR title?

Comment (dlnr) [reply packages/react/src/Spotlight/Spotlight.test.tsx:53]: Allow ‘Grid Cell’ and 'Spotlight' to render a custom HTML element

Comment (VincentSmedinga) [reply packages/react/src/Spotlight/Spotlight.test.tsx:53]: No, Spotlight didn’t change here. It had an `as` prop from the start in #636, you created it yourself.

**Pull request merged**